agoraStreaming/src/Components/SwitchVideo/styles.ts

```diff
@@ -0,0 +1,15 @@
+import {StyleSheet, ViewStyle} from 'react-native';
+
+type SwitchVideoType = {
```

Comment:
SwitchVideoStyles or SwitchVideoStyleTypes, cause you have the same names in styles and component types